spec/classes/pulp_apache_spec.rb

```diff
@@ -188,19 +207,10 @@
 <Directory /var/www/pub/http/isos >
     Options FollowSymLinks Indexes
 </Directory>
-
-
-# -- GPG Keys -------------------
-Alias /pulp/gpg /var/www/pub/gpg
-
-<Directory /var/www/pub/gpg/>
-    Options FollowSymLinks Indexes
-</Directory>
 ')
 
-        is_expected.to contain_file('/etc/pulp/vhosts80/rpm.conf').with(
+        is_expected.to contain_file('/etc/pulp/vhosts80/iso.conf').with(
         :content => 'Alias /pulp/isos /var/www/pub/http/isos
```

Comment:
It might be slightly easier to read if you use ```"Alias /pulp/isos /var/www/pub/http/isos\n"```

Reply:
agreed, thanks!